Documentation/contributor-guide/release.md

```diff
@@ -78,11 +78,14 @@ which don't need to be executed before releasing each version.
 
    It generates all release binaries under the directory `/tmp/etcd-release-${VERSION}/etcd/release/` and images. Binaries are pushed to the Google Cloud bucket
    under project `etcd-development`, and images are pushed to `quay.io` and `gcr.io`.
+
+   **Remove the `quay.io` login entry from `~/.docker/config.json` after pushing Docker images.**
 7. Publish the release page on GitHub
    - Set the release title as the version name
    - Choose the correct release tag (generated from step #4)
    - Follow the format of previous release pages
    - Attach the generated binaries and signature file
+   - Verify the historical binary size for each architecture. If there's a big difference, verify that it works for that architecture
```

Comment:
I think longer term we could have a workflow trigger either automatically or manually during a release window which performs artifact validations.  

For now let's just add this note about manual verification though.

Reply:
I would love to run e2e tests on the generated artifacts.

Reply:
I completely agree. This is a short-term solution while we implement better automation.